controller/structures.xml

```diff
@@ -23,19 +23,16 @@
                     </action>
                     <action id="group-properties" name="Properties"  url="/taoGroups/Groups/editGroup" group="content" context="instance">
                         <icon id="icon-edit"/>
-                    </action> 
-                    <action id="group-filter" name="Filter" context="*" group="search" binding="filter">
-                        <icon id="icon-filter"/>
-                    </action>
-                    <action id="group-search" name="Search" url="/taoGroups/Groups/search" context="*" group="search" binding="launchFinder">
-                        <icon id="icon-find"/>
                     </action>
                     <action id="group-class-new" name="New class" url="/taoGroups/Groups/addSubClass" context="resource" group="tree" binding="subClass">
                         <icon id="icon-folder-open"/>
                     </action>
                     <action id="group-delete" name="Delete" url="/taoGroups/Groups/delete" context="resource" group="tree" binding="removeNode">
                         <icon id="icon-bin"/>
                     </action>
+                    <action id="group-delete-all" name="Delete" url="/taoGroups/Groups/deleteAll" context="resource" multiple="true" group="tree" binding="removeNodes">
```

Comment:
The display name of 'delete all' is the same as for the 'delete one'.

Reply:
They aren't supposed to be displayed at the same time, multiple and single contexts are exclusives. The bug is fixed in tao-core